source/_components/light.flux_led.markdown

```diff
@@ -89,3 +90,11 @@ Will add a light with white mode (default). Changing the brightness will set the
       name: NAME
       mode: "rgbw"
 ```
+
+Some devices such as the Ledenet RGBW controller use a slightly differenct protocol for communicating the brightness to each color channel. If your device is only turning on or off but not changing color or brightness try adding the LEDENET protocol.
+
+```yaml
```

Comment:
Should we include the full configuration snippet to avoid user's error (including the platform statement?)